lib/lnc.ts

```diff
@@ -78,6 +78,38 @@ export default class LNC {
         );
     }
 
+    get status() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientStatus &&
+            this.wasm.wasmClientStatus()
+        );
+    }
+
+    get expiry() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientGetExpiry &&
+            this.wasm.wasmClientGetExpiry()
+        );
+    }
+
+    get hasPerms() {
+        return (
+            this.wasm &&
+            this.wasm.wasmClientHasPerms &&
+            this.wasm.wasmClientHasPerms()
```

Comment:
`wasmClientHasPerms()` accepts a string parameter specifying the RPC method
See the example page https://github.com/lightninglabs/lightning-node-connect/blob/master/example/index.html#L196